kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/core/util/ReflectionUtil.java

```diff
@@ -92,4 +92,33 @@ public static Object invokeMethod(final Object target, final String methodName,
         method.setAccessible(true);
         return method.invoke(target, parameterValues);
     }
+    
+    /**
+     * invoke method in parent class.
+     *
+     * @param target target object
+     * @param methodName method name
+     * @param parameterTypes parameter types
+     * @param parameterValues parameter values
+     * @return invoke method result.
+     * @throws NoSuchMethodException no such field exception
+     * @throws InvocationTargetException invocation target exception
+     * @throws IllegalAccessException illegal access exception
+     */
+    public static Object invokeMethodInParentClass(final Object target, final String methodName, final Class<?>[] parameterTypes,
+                                                   final Object[] parameterValues) throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
```

Comment:
`invokeMethodInParentClass` is just used in unit test, could we put it to test package?
Or else could we add unit test for it

Reply:
`ReflectionUtil` only used for test, but if move it to `src/test` and using `test-jar`, need to add dependencies in multiple modules, so I just add unit test now

Reply:
OK